
时间：2017-06-27 09:24:09,247 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：26073 
System.InvalidOperationException: 超时时间已到。超时时间已到，但是尚未从池中获取连接。出现这种情况可能是因为所有池连接均在使用，并且达到了最大池大小。
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Data.x_dict.GetDictList(String code, String upval, DataClassesDataContext db) 位置 F:\HT\xue\src\Xc\Data\DataClasses.cs:行号 71
   在 X.Web.XFace.GetDictList(String code, String upval) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 175
   在 X.App.Views._web.getSubject(Int32 age) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 48
   在 X.App.Views._web.InitView() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 25
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 09:24:25,594 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：42420 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Data.x_dict.GetDictList(String code, String upval, DataClassesDataContext db) 位置 F:\HT\xue\src\Xc\Data\DataClasses.cs:行号 71
   在 X.Web.XFace.GetDictList(String code, String upval) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 175
   在 X.App.Views._web.getSubject(Int32 age) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 48
   在 X.App.Views._web.InitView() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 25
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-06-27 09:41:28,287 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：104949 

时间：2017-06-27 09:41:42,053 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：118714 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 09:41:42,053 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：118714 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 09:42:15,898 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：152559 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 09:42:15,899 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：152560 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:04:03,330 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8371978 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:04:06,157 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8374806 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:04:40,268 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8408916 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:05:01,157 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8429806 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:08:56,641 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8665289 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:10:09,418 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8738066 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:12:08,026 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8856674 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:12:24,722 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8873370 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:12:40,225 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8888873 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:13:44,768 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8953417 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:13:47,702 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8956350 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:20:05,179 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：9333827 

时间：2017-06-27 15:20:13,897 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：372 

时间：2017-06-27 15:20:26,272 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：12747 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:20:26,272 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：12747 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:21:43,670 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：90145 

时间：2017-06-27 15:26:32,634 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：1357 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.question.InitDict() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\question.cs:行号 50
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:26:32,707 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1430 

时间：2017-06-27 15:29:49,477 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：3752 

时间：2017-06-27 15:30:02,190 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：16465 
System.NotSupportedException: 方法“System.String ToString(System.String)”不支持转换为 SQL。
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.SqlSelectionSkipper.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitNew(SqlNew sox)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Linq.IQueryProvider.Execute[S](Expression expression)
   在 System.Linq.Queryable.Count[TSource](IQueryable`1 source)
   在 X.App.Apis.mgr.admin.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\admin\list.cs:行号 35
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:31:02,396 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：76671 
System.NotSupportedException: 方法“System.String ToString(System.String)”不支持转换为 SQL。
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.SqlSelectionSkipper.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitNew(SqlNew sox)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Linq.IQueryProvider.Execute[S](Expression expression)
   在 System.Linq.Queryable.Count[TSource](IQueryable`1 source)
   在 X.App.Apis.mgr.admin.list.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:33:00,728 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：195004 
System.NotSupportedException: 方法“System.String ToString(System.String)”不支持转换为 SQL。
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.SqlSelectionSkipper.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitNew(SqlNew sox)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Linq.IQueryProvider.Execute[S](Expression expression)
   在 System.Linq.Queryable.Count[TSource](IQueryable`1 source)
   在 X.App.Apis.mgr.admin.list.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:34:03,718 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：4890 

时间：2017-06-27 15:34:09,590 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：10762 

时间：2017-06-27 15:34:31,015 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：32186 
System.NotSupportedException: 方法“System.String ToString(System.String)”不支持转换为 SQL。
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.SqlSelectionSkipper.VisitColumn(SqlColumn col)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitNew(SqlNew sox)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Linq.IQueryProvider.Execute[S](Expression expression)
   在 System.Linq.Queryable.Count[TSource](IQueryable`1 source)
   在 X.App.Apis.mgr.admin.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\admin\list.cs:行号 36
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:37:02,123 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：4085 

时间：2017-06-27 15:37:08,240 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：10201 

时间：2017-06-27 15:42:53,023 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：8424 

时间：2017-06-27 15:46:25,726 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：5429 

时间：2017-06-27 15:46:44,352 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：24055 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.admin.list.<>c.<Execute>b__12_0(x_mgr o) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\admin\list.cs:行号 25
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Json.Serialization.JsonArrayContract.CreateWrapper(Object list)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.SerializeInternal(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonConvert.SerializeObject(Object value, Formatting formatting, JsonSerializerSettings settings)
   在 X.Json.JsonConvert.SerializeObject(Object value, JsonSerializerSettings settings)
   在 X.Core.Utility.Serialize.ToJson(Object o, Boolean ignore) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 26
   在 X.Core.Utility.Serialize.ToJson(Object o) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 18
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:47:26,440 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：66143 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.admin.list.<>c.<Execute>b__12_0(x_mgr o) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\admin\list.cs:行号 25
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Json.Serialization.JsonArrayContract.CreateWrapper(Object list)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.SerializeInternal(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonConvert.SerializeObject(Object value, Formatting formatting, JsonSerializerSettings settings)
   在 X.Json.JsonConvert.SerializeObject(Object value, JsonSerializerSettings settings)
   在 X.Core.Utility.Serialize.ToJson(Object o, Boolean ignore) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 26
   在 X.Core.Utility.Serialize.ToJson(Object o) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 18
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:48:32,260 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：8911 

时间：2017-06-27 15:49:51,779 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：3869 

时间：2017-06-27 15:49:57,644 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：9733 

时间：2017-06-27 15:50:02,511 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：14600 

时间：2017-06-27 15:50:12,481 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：24570 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.admin.list.<>c.<Execute>b__12_0(x_mgr o) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\admin\list.cs:行号 25
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Json.Serialization.JsonArrayContract.CreateWrapper(Object list)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.SerializeInternal(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonConvert.SerializeObject(Object value, Formatting formatting, JsonSerializerSettings settings)
   在 X.Json.JsonConvert.SerializeObject(Object value, JsonSerializerSettings settings)
   在 X.Core.Utility.Serialize.ToJson(Object o, Boolean ignore) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 26
   在 X.Core.Utility.Serialize.ToJson(Object o) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 18
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 15:51:23,400 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：95489 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.admin.list.<>c.<Execute>b__12_0(x_mgr o)
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Json.Serialization.JsonArrayContract.CreateWrapper(Object list)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.SerializeValue(JsonWriter writer, Object value, JsonContract valueContract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerProperty)
   在 X.Json.Serialization.JsonSerializerInternalWriter.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.SerializeInternal(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonSerializer.Serialize(JsonWriter jsonWriter, Object value)
   在 X.Json.JsonConvert.SerializeObject(Object value, Formatting formatting, JsonSerializerSettings settings)
   在 X.Json.JsonConvert.SerializeObject(Object value, JsonSerializerSettings settings)
   在 X.Core.Utility.Serialize.ToJson(Object o, Boolean ignore) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 26
   在 X.Core.Utility.Serialize.ToJson(Object o) 位置 F:\HT\xue\src\Xc\Core\Utility\Serialize.cs:行号 18
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-27 16:53:39,058 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：9622 

时间：2017-06-27 17:06:33,111 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：249454 

时间：2017-06-27 17:06:34,689 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：251032 

时间：2017-06-27 17:49:20,655 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T试卷不存在)","code":"T试卷不存在"} 
时间：3680 

时间：2017-06-27 17:49:28,399 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T试卷不存在)","code":"T试卷不存在"} 
时间：11423 
